### PR TITLE
Sort libraries by GitHub stars

### PR DESCRIPTION
**Pull request by rictic**
Also, since we now need a GH repo for every library, make the name a link to the repo

On the plus side, this means that the info a person is most likely to be curious about will be right at the top

On the minus side, if the library they want to know about isn't near the top, they'll pretty much have to ctrl-f to find it / know whether we have an entry for it or not

### CODE REVIEW

**Comment by rictic** [comment]
Related to #1140

File: docs/index.handlebars [hunk 1]
```diff
@@ -116,7 +116,8 @@ limitations under the License.
             <div class="level-item has-text-centered">
               <div>
                 <p class="heading">Library</p>
-                <h3 class="title">{{fullName}} <span class="library-version">{{results.libraryVersion}}</span></h3>
+                <h3 class="title"><a href="{{repo.html_url}}">{{fullName}}</a> <span
+                    class="library-version">{{results.libraryVersion}}</span></h3>
```
Comment (AndrewJakubowicz) [inline docs/index.handlebars:120]: Nit and non actionable here. I wonder if there is some way to shorten this or link this to the github version. For example this is very long on React experimental.

Maybe even truncate it with css?w

File: index.js [hunk 2]
```diff
@@ -119,6 +122,14 @@ async function runTests() {
   return failed;
 }
 
+async function fetchMetadata(library) {
+  const repoMeta = (await (await fetch(`https://api.github.com/repos/${library.packageJson.library_repo}`)).json());
```
Comment (AndrewJakubowicz) [inline index.js:126]: TIL!

**Review by AndrewJakubowicz — APPROVED** [review]
LGTM! Really great improvement!

I went through the repo links and checked them manually. The list feels more intuitive as you scroll down now.

**Pull request merged**